src/components/ChangePlan/PlanCalculator.js

```diff
@@ -29,24 +29,42 @@ const NavigatorTabs = ({ tabs, pathType, selectedPlanType }) => {
 const Discounts = ({ discountsList, handleChange }) => {
   const [selectedDiscount, setSelectedDiscount] = useState(discountsList[0]);
   return (
-    <div style={{ marginBottom: '40px' }}>
-      {discountsList.map((discount, index) => (
-        <button
-          key={index}
-          style={{
-            padding: '10px',
-            border: '1px solid #000',
-            backgroundColor: discount.id === selectedDiscount.id ? '#33ad73' : '#f6f6f6',
-          }}
-          onClick={() => {
-            handleChange(discount);
-            setSelectedDiscount(discount);
-          }}
-        >
-          {discount.description}
-        </button>
-      ))}
-    </div>
+    <>
+      <div className="dp-wrap-subscription">
+        <h4>Suscripcion</h4>
```

Comment:
Please add the language variable, so when we have the real text is easier to change

Reply:
The same for all loose texts...

Reply:
What do you think about add this changes in other PR. There are some texts that aren't so easy to apply because, for example, can change based on what type of plan do you select or depends on variable data.

Reply:
You owe this change since some PRs... There's no hurry here, you can make this texts changes in another commit perhaps.